lib/membrane_live_web/channels/event_channel.ex

```diff
@@ -618,7 +558,9 @@ defmodule MembraneLiveWeb.EventChannel do
         {:error, :webinar_finished}
 
       socket.assigns.is_moderator ->
-        Event.start(socket.assigns.event_id, name: {:global, socket.assigns.event_id})
+        result = Room.start(socket.assigns.event_id, name: {:global, socket.assigns.event_id})
+        EventController.room_created(socket.assigns.event_id)
+        result
```

Comment:
I didn't put much thought to it, but maybe the `EventController.room_created` should be responsible for calling `Room.start` (and change name to something appropriate)? It seems like its the only function from `Room` called in `event_channel.ex`, so maybe `EventController should be kind of a proxy between `EventChannel` and `Room`?